tests/unit/xoopsLib/Xoops/Core/HttpRequestTest.php

```diff
@@ -42,15 +42,12 @@ protected function tearDown()
 
     /**
      * @covers Xoops\Core\HttpRequest::getInstance
-     * @todo   Implement testGetInstance().
      */
     public function testGetInstance()
     {
-
-        $instance = HttpRequest::getInstance();
+        $instance = $this->object;
```

Comment:
This code was testing that the HttpRequest was functioning as a singleton instance.